controllers/controllers/cluster_controller.go

```diff
@@ -36,15 +36,16 @@ func NewClusterReconciler(client client.Client, log logr.Logger, scheme *runtime
 		log:       log,
 		validator: validator,
 		defaulter: defaulter,
+		tracker:   tracker,
 	}
 }
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&anywherev1.Cluster{}).
-		Watches(&source.Kind{Type: &anywherev1.VSphereDatacenterConfig{}}, &handler.EnqueueRequestForObject{}).
-		Watches(&source.Kind{Type: &anywherev1.VSphereMachineConfig{}}, &handler.EnqueueRequestForObject{}).
+		// Watches(&source.Kind{Type: &anywherev1.VSphereDatacenterConfig{}}, &handler.EnqueueRequestForObject{}).
+		// Watches(&source.Kind{Type: &anywherev1.VSphereMachineConfig{}}, &handler.EnqueueRequestForObject{}).
```

Comment:
These were commented out because they are triggering reconcile for the EKS-A-cluster with the wrong namespace/name (using the one defined in VSphereDatacenter and VSpherMachineConfig)

Two new functions will be written to trigger the reconcile on the EKS-A cluster correctly 

```
DataCenterToClusterMapFunc and MachineToClusterMapFunc
```

But that will be done on a separate PR.

Reply:
Maybe just remove then?